distribution/kernel/carbon-home/repository/resources/conf/default.json

```diff
@@ -244,5 +244,6 @@
   "database.registry_db.pool_options.testOnBorrow" : "true",
   "database.registry_db.pool_options.validationQuery" : "SELECT 1",
   "database.registry_db.pool_options.validationInterval" : "30000",
-  "database.registry_db.pool_options.defaultAutoCommit" : "true"
+  "database.registry_db.pool_options.defaultAutoCommit" : "true",
```

Comment:
Different configs compare to 5.11.0 PR. Any reason?

Reply:
More secure option is `"admin_console.resolve_absolute_urls.enable": true`, but we decided to make this false in 5.11 to not to introduce any behavioral changes.